cvat/scripts/pascal_voc.py

```diff
@@ -0,0 +1,42 @@
+import os
+import argparse
+import xml.etree.ElementTree
+from PIL import Image
+from pascal_voc_writer import Writer
+
+
+def process_cvat_xml(xml_file, img_dir, annotation_dir):
```

Comment:
@syonekura,  Thank you for the MR! I see that the script supports only "annotation" format and doesn't support "interpolation" format. What is the reason for that?

Reply:
As I understand from [XML Format](https://github.com/opencv/cvat/blob/master/cvat/apps/documentation/xml_format.md#interpolation), the interpolation format doesn't have a image file for each object (is a per video annotation), and it doesn't declare the path of the video file. 

I'm not sure if this script should create not only the VOC XML's but also the image files from the video. If it should create them, is there an easy way to get the path of the video, or query for a specific frame of the task? If it shouldn't create the image files (for example if CVAT already let you download the interpolations as images), I still need the 'path' of those future images, is there a way to query the map between the frame number and the image filename?

Reply:
@syonekura ,

Please be sure that the script will handle unsupported annotation files (like for interpolation) gracefully.